python/dask_cudf/dask_cudf/core.py

```diff
@@ -235,17 +235,27 @@ def sort_values(
         set_divisions=False,
         ascending=True,
         na_position="last",
+        sort_function=None,
+        sort_function_kwargs=None,
```

Comment:
These kwargs are already implemented upstream (and documented in the docstring), correct?

Reply:
Yes, here are the docstrings if you're interested:

https://github.com/dask/dask/blob/c2b962fec1ba45440fe928869dc64cfe9cc36506/dask/dataframe/core.py#L4345-L4351